packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -340,7 +343,10 @@ function ActiveRequests({ votingAccount, votingGateway }) {
       if (!checkboxesChecked[index] || !editState[index]) {
         continue;
       }
-      const price = web3.utils.toWei(editState[index]);
+      const identifierPrecision = IDENTIFIER_NON_18_PRECISION[hexToUtf8(pendingRequests[index].identifier)]
```

Comment:
nit: maybe it makes sense to put a function (like `getIdentifierPrecision`) in PriceIdentifierUtils that will do this defaulting logic for you? Just because we seem to have this same code in a few places.

Reply:
Agreed, made the change